node-tests/blueprints/controller-test-test.js

```diff
@@ -92,4 +157,18 @@ describe('Blueprint: controller-test', function() {
       });
     });
   });
+
+  describe('in addon - module unification', function() {
+    beforeEach(function() {
+      return emberNew().then(() => fs.ensureDirSync('src'));
+    });
+
+    it('controller-test foo', function() {
+      return emberGenerateDestroy(['controller-test', 'foo'], _file => {
+        expect(_file('src/ui/routes/foo/controller-test.js')).to.equal(
+          fixture('controller-test/default.js')
+        );
+      });
+    });
+  });
```

Comment:
perhaps it would be good to add a test for a nested controller tests for addons and apps for both MU and classic? we seem to be missing those test cases